app/src/main/java/com/example/tools/timber/CrashlyticsTree.java

```diff
@@ -18,7 +18,7 @@ protected boolean isLoggable(String tag, int priority) {
 
     @Override
     protected void log(int priority, @Nullable String tag, @Nullable String message, @Nullable Throwable throwable) {
-        if (priority == Log.ASSERT) {
+        if (isLoggable(tag, priority)) {
```

Comment:
As the tag is @Nullable, you would need to make the method isLoggable as this:

```
    @Override
    protected boolean isLoggable(@Nullable String tag, int priority) {
        return priority == Log.ASSERT;
    }
```
otherwise findbugs static analysis tool fails